src/components/ebay-lightbox-dialog/index.marko

```diff
@@ -54,25 +41,6 @@ $ var handleLabel = state.expanded ? a11yMinimizeText : a11yMaximizeText;
     window-class=[
         "lightbox-dialog__window--animate",
         isMini && "lightbox-dialog__window--mini",
-        state.expanded && "lightbox-dialog__window--expanded"
     ]>
-        <@top>
-            <button
-                aria-label=handleLabel
-                type="button"
-                class=`lightbox-dialog__handle`
-                onClick("handleExpand")
-                onTouchstart("handleTouchStart")
-                onTouchmove("handleTouchMove")
-                onTouchend("handleTouchEnd")
-                onTouchcancel("handleTouchEnd")
-            ></button>
-            <if(bannerImgSrc)>
```

Comment:
The `bannerImgSrc` shouldn't be removed I suppose no?

Reply:
Good catch!